crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -218,6 +218,7 @@ pub trait EthTransactions: Send + Sync {
             + Send;
 }
 
+// todo: add header td call
```

Comment:
im removing this todo - not sure what it's about, we can fix in followup if it's relevant still